types/src/on_chain_config/consensus_config.rs

```diff
@@ -157,36 +157,27 @@ impl Default for ConsensusConfigV1 {
 }
 
 #[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
-pub struct ConsensusConfigV1Ext {
-    pub main: ConsensusConfigV1,
-    pub extra_feature_flags: Vec<bool>,
-}
-
-#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
-#[allow(non_camel_case_types)]
-pub enum ConsensusExtraFeature {
-    ProposalWithValidatorTransactions = 0,
+pub struct ConsensusExtraFeatures {
+    features: Vec<bool>,
```

Comment:
Why a vector of bool? what are the features?

Reply:
It's a vec so it's extensible like the onchain feature vec. Currently there's only 1 feature.

Reply:
the index you use will be a `ConsensusExtraFeature` enum (similar to `aptos_types::on_chain_config::aptos_features::FeatureFlag` today)

To enable, you update consensus config via a governance proposal where the config is serialized.

Reply:
I see. Maybe add a comment here?

Reply:
added.